src/main/resources/templates/update_user.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
Couldn't you use the same HTML file as Create User page for updating a user?
Definitely, you can by making some changes to it.
Remove this file and reuse the Create user html file.